README.md

```diff
@@ -147,6 +155,51 @@ Then, visit the URL logged to your terminal to view the now locally-running web
 
 You can stop the local PostgreSQL container server by running `docker-compose down`. If you wish to completely wipe the container database, including all the data added by the migrations, run `docker volume rm digital_marketplace_dm-vol`.
 
+#### Containerized
```

Comment:
Okay, after testing locally and realizing that this vs code approach seems to be dependent on the version of VS code being used I have a suggestion.  Since none of us seem to be using docker containers for local app development let's take out all the vs code specific stuff from the readme and just present the Docker commands as a way of running the project locally, but making in clear it's not suitable for app development something like:
"If a local environment needs to be spun up to demo or test the app, the commands `docker-compose build` followed by `docker-compose up`  will build and deploy the containerized app locally. However changes to the codebase will not be reflected in the app until the build command is re-run. This slow turn around makes local development using docker a less desirable approach."

Reply:
Thanks, this seems like a good solution